RiotSharp/Endpoints/ThirdPartEndpoint/ThirdPartyEndpoint.cs

```diff
@@ -19,15 +19,16 @@ public ThirdPartyEndpoint(IRateLimitedRequester requester)
 
         public string GetThirdPartyCodeBySummonerId(Region region, long summonerId)
         {
-            return _requester.CreateGetRequest(ThirdPartyRootUrl + 
-                string.Format(ThirdPartyBySummonerUrl, summonerId),
-                region);
+            return _requester
+                .CreateGetRequest(ThirdPartyRootUrl + string.Format(ThirdPartyBySummonerUrl, summonerId), region)
+                .Replace("\"", "");
```

Comment:
I think the `Replace` is not needed

Reply:
I used replace in order to improve readability you mentioned here  

> You might want to change this into a string literal to improve readability 

If we remove if our two new test will fail for this case. 
```
Assert.AreEqual failed. Expected:<AD irelia (AS)>. Actual:<"AD irelia (AS)">
```

Reply:
What if somebody has a quote character in their 3d party code?

Reply:
Riot returns a string with quotation marks, e.g. `"AD irelia (AS)"` . We could leave it as it is. However, it might be confusing to use. Just remove the quotation marks at the beginning and at the end and make sure it's documented in the interface method.

Reply:
A simple solution would be:
```
	var rawResponse = _requester.CreateGetRequest(ThirdPartyRootUrl + 
		string.Format(ThirdPartyBySummonerUrl, summonerId), region);
	return rawResponse.Substring(1, rawResponse.Length - 2);
```

Reply:
we might want to check the prefix and suffix of the string before removing characters

Reply:
The response content is still technically JSON (a json string literal) so you should still be able to use JSON.net to parse it.

Double quotes will end up as `\"` and unicode may (or may not be) encoded as `\u00f8` and still be valid JSON.